apps/wallet/src/ui/app/pages/accounts/AddAccountPage.tsx

```diff
@@ -5,22 +5,28 @@ import { LedgerLogo17 as LedgerLogo } from '@mysten/icons';
 import { useState, type ReactNode } from 'react';
 import toast from 'react-hot-toast';
 import { useNavigate, useSearchParams } from 'react-router-dom';
+import Browser from 'webextension-polyfill';
 import { useAccountsFormContext } from '../../components/accounts/AccountsFormContext';
 import { ConnectLedgerModal } from '../../components/ledger/ConnectLedgerModal';
 import { getLedgerConnectionErrorMessage } from '../../helpers/errorMessages';
+import { useAppSelector } from '../../hooks';
+import { AppType } from '../../redux/slices/app/AppType';
 import { SocialButton } from '../../shared/SocialButton';
 import { Button } from '_app/shared/ButtonUI';
 import { Text } from '_app/shared/text';
 import Overlay from '_components/overlay';
 import { ampli } from '_src/shared/analytics/ampli';
 
 export function AddAccountPage() {
-	const [isConnectLedgerModalOpen, setConnectLedgerModalOpen] = useState(false);
 	const [searchParams] = useSearchParams();
 	const navigate = useNavigate();
 	const sourceFlow = searchParams.get('sourceFlow') || 'Unknown';
 	const showSocialSignInOptions = sourceFlow !== 'Onboarding';
+	const forceShowLedger =
+		searchParams.has('showLedger') && searchParams.get('showLedger') !== 'false';
```

Comment:
is the `searchParams.get('showLedger') !== 'false'` check needed?

Reply:
not really because we never set it to false but I was overthinking about manually adding it to the url